src/pybind/mgr/dashboard/tests/test_erasure_code_profile.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from .. import mgr
+from .helper import ControllerTestCase
+from ..controllers.erasure_code_profile import ErasureCodeProfile
+
+mock_osd_map = {
+    'erasure_code_profiles': {
+        'test': {
+            'k': '2',
+            'm': '1'
+        }
+    }
+}
+
+
+class ErasureCodeProfileTest(ControllerTestCase):
+    @classmethod
+    def setup_server(cls):
+        mgr.get.side_effect = lambda key: {
+            'osd_map': mock_osd_map,
```

Comment:
Because ``mock_osd_map`` is not used anywhere else i suggest to apply this patch:

```
--- src/pybind/mgr/dashboard/tests/test_erasure_code_profile.py	(date 1540706579000)
+++ src/pybind/mgr/dashboard/tests/test_erasure_code_profile.py	(date 1540818956000)
@@ -4,21 +4,19 @@
 from .helper import ControllerTestCase
 from ..controllers.erasure_code_profile import ErasureCodeProfile
 
-mock_osd_map = {
-    'erasure_code_profiles': {
-        'test': {
-            'k': '2',
-            'm': '1'
-        }
-    }
-}
-
 
 class ErasureCodeProfileTest(ControllerTestCase):
     @classmethod
     def setup_server(cls):
         mgr.get.side_effect = lambda key: {
-            'osd_map': mock_osd_map,
+            'osd_map': {
+                'erasure_code_profiles': {
+                    'test': {
+                        'k': '2',
+                        'm': '1'
+                    }
+                }
+            },
             'health': {'json': '{"status": 1}'},
             'fs_map': {'filesystems': []},
```

Reply:
@votdev Github offers the feature to insert suggestions. Those can be immediately applied by the author and are therefore much easier to apply than a patch. Please consider using that feature if applicable.